README.md

```diff
@@ -1,131 +1,102 @@
-relative measures over long-term data. For example, the leaderboard only inspects the last 30 days
-of contributions, allowing new contributors to get to the top more easily.
+Available at https://project-dashboard.18f.gov/default:
 
-Preview: The [SilverStripe CMS](http://silverstripe.org) project, aggregating over 50 repositories
-that the project either maintains or actively contributes to.
-View it live at [github-dashing.herokuapp.com](http://github-dashing.herokuapp.com/default).
-![Preview](assets/images/preview.png?raw=true)
+- [Travis CI](http://travis-ci.org) build status (updates every 2 minutes)
+- Repos whose `.about.yml` defines them as testable but don't have builds
+	running on a CI service such as Travis or CircleCI (updates once a day)
+- Repos that don't have an `.about.yml` (updates once a day)
 
-Preview: Pull request stats with trend projection for current month
-![Preview](assets/images/preview_stats.png?raw=true)
+![Preview](assets/images/project_dashboard_default_preview.png?raw=true)
 
-Preview: Travis build status with per-branch status and code quality indicators
-![Preview](assets/images/preview_travis.png?raw=true)
+Available at https://project-dashboard.18f.gov/github_stats:
 
-The dashboard is based on [Dashing](http://shopify.github.com/dashing), a Ruby web application
-built on the [Sinatra](http://www.sinatrarb.com) framework. It uses the Github API rather than
-[githubarchive.org](http://githubarchive.org) data dumps because of the immediate nature
-of dashboard update (refreshes every hour by default). The code used to be based
-on Google BigQuery aggregation, but this turned out to be infeasible due to query size and BigQuery pricing.
+- Trend projections for current month on issues opened, issues closed and pull requests
 
-## Setup
+![Preview](assets/images/project_dashboard_github_stats_preview.png?raw=true)
 
-### Generic Configuration
+The dashboard is based on [Dashing](http://shopify.github.com/dashing), a Ruby
+web application built on the [Sinatra](http://www.sinatrarb.com) framework.
 
-First install the required dependencies through `bundle install`.
+APIs used:
 
-The project is configured through environment variables.
-Copy the `.env.sample` configuration file to `.env`.
+- [GitHub](https://developer.github.com/)
+- [Travis](http://docs.travis-ci.com/api/)
+- [18F Team API/Projects](https://team-api.18f.gov/public/api/projects/)
 
-All environment variables are optional, apart from `ORG`.
+## Setup
 
- * `ORG`: GitHub organization.
-   Example: `18F`
- * `SINCE`: Date string, or relative time parsed through [http://guides.rubyonrails.org/active_support_core_extensions.html](ActiveSupport). Example: `1.month.ago.beginning_of_month`, `2012-01-01`
- * `GITHUB_OAUTH_TOKEN`: Required in order to avoid being rate limited.
+    git clone https://github.com/18F/github-dashing.git && cd github-dashing
+    script/bootstrap
```

Comment:
I forget where I first saw the use of `script/bootstrap`. It might have been [Octokit](https://github.com/octokit/octokit.rb) 2 or 3 years ago. Since then, that's what I've always used, and I think I've seen others use the same. Then when Rails introduced `bin/setup` in 4.2 I think, I've been seeing that in some Rails projects. Then you also have @mbland championing the `go` prefix.

In the end, I don't think the name matters, as long as there's a script that makes life easier.